### PR TITLE
Update RELEASE_ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by ajnavarro**

### CODE REVIEW

File: docs/RELEASE_ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -36,18 +36,19 @@ For each RC published in each stage:
 
 - version string in `version.go` has been updated (in the `release-vX.Y.Z` branch).
 - new commits should be added to the `release-vX.Y.Z` branch from `master` using `git cherry-pick -x ...`
+- `release-` branches are protected. You can do all needed updates on a separated branch and when everything is settled push to `release-vX.Y.Z`
```
Comment (SgtPooki) [inline docs/RELEASE_ISSUE_TEMPLATE.md:39]: should this be a subitem of the above bullet-point?

**Comment by BigLep** [comment]
Per 2022-08-22 discussion, the list in https://github.com/ipfs/kubo/blob/master/docs/EARLY_TESTERS.md should be updated (I don't know what to).

**Comment by ajnavarro** [comment]
@BigLep updating early testers list here: https://github.com/ipfs/kubo/pull/9218

File: docs/RELEASE_ISSUE_TEMPLATE.md (hunk 1 above)
Comment (ajnavarro) [reply docs/RELEASE_ISSUE_TEMPLATE.md:39]:
```suggestion
  - `release-` branches are protected. You can do all needed updates on a separated branch and when everything is settled push to `release-vX.Y.Z`
```

File: docs/RELEASE_ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -70,7 +71,7 @@ Checklist:
 - [ ] **Stage 1 - Internal Testing**
   - [ ] CHANGELOG.md has been updated
     - use [`./bin/mkreleaselog`](https://github.com/ipfs/go-ipfs/tree/master/bin/mkreleaselog) to generate a nice starter list
```
Comment (ajnavarro) [inline docs/RELEASE_ISSUE_TEMPLATE.md:73]: `mkreleaselog` is not working. It expects the source code to be inside $GO_HOME/src, and also needs zsh to work properly. We can change this to use the `Generate release notes` button here: https://github.com/ipfs/kubo/releases/edit/v0.15.0-rc1

File: docs/RELEASE_ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -70,7 +71,7 @@ Checklist:
...
-  - [ ] Infrastructure Testing:
+  - [ ] Infrastructure Testing. Open an issue against https://github.com/protocol/bifrost-infra like https://github.com/protocol/bifrost-infra/issues/2046 but spell out all that we want (gateways, bootstrapper, and cluster)
     - [ ] Deploy new version to a subset of Bootstrappers
     - [ ] Deploy new version to a subset of Gateways
     - [ ] Deploy new version to a subset of Preload nodes
```
Comment (ajnavarro) [inline docs/RELEASE_ISSUE_TEMPLATE.md:77]: `Collect metrics every day`: Specify how to collect these metrics.

File: docs/RELEASE_ISSUE_TEMPLATE.md (hunk 2 above)
Comment (ajnavarro) [inline docs/RELEASE_ISSUE_TEMPLATE.md:73]: How I make it works:

- install zsh
- GO111MODULE=off go get -u github.com/ipfs/kubo
- Go to $GOPATH/src/ipfs/kubo and execute `./bin/mkreleaselog`

**Comment by BigLep** [comment]
2022-08-25:
* Need to have a mechanism for notifying bifrost sooner about upcoming release
* Need to clarify the steps for reviewing Grafana dashboard: https://protocollabs.grafana.net/d/8zlhkKTZk/gateway-slis-precomputed?orgId=1
* Add a step to put ipfs-support-providers on high alert when in the RC phase.

File: docs/RELEASE_ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -80,12 +81,17 @@ Checklist:
       - [ ] Ensure the RC is published to [the NPM package](https://www.npmjs.com/package/go-ipfs?activeTab=versions) ([happens automatically, just wait for CI](https://github.com/ipfs/npm-go-ipfs/actions))
       - [ ] Upgrade to the RC in [ipfs-desktop](https://github.com/ipfs-shipyard/ipfs-desktop) and push to a branch ([example](https://github.com/ipfs/ipfs-desktop/pull/1826/commits/b0a23db31ce942b46d95965ee6fe770fb24d6bde)), and open a draft PR to track through the final release ([example](https://github.com/ipfs/ipfs-desktop/pull/1826))
       - [ ] Ensure CI tests pass, repeat for new RCs
-    - [ ] [IPFS Companion](https://github.com/ipfs-shipyard/ipfs-companion) - @lidel
+    - [ ] [IPFS Companion](https://github.com/ipfs-shipyard/ipfs-companion)
+      - Start kubo daemon of the version to release.
+      - Start a fresh chromium or chrome instance using `chromium --user-data-dir=$(mktemp -d)`
+      - Start a fresh firefox instance using `firefox --profile $(mktemp -d)`
+      - Install IPFS companion for the specific shop.
```
Comment (lidel) [inline docs/RELEASE_ISSUE_TEMPLATE.md:88]:
```suggestion
- Install IPFS Companion from [vendor-specific store](https://github.com/ipfs/ipfs-companion/#readme).
```

**Comment by BigLep** [comment]
2022-08-31 items from @ajnavarro and @BigLep discussion:
1. Add a link from release notes to Discuss post (like we did here: https://github.com/ipfs/kubo/releases/tag/v0.15.0 )
2. Keep the release notes as trim as possible (removing some of the top headers, like we did here: https://github.com/ipfs/kubo/releases/tag/v0.15.0 )

**Comment by ajnavarro** [comment]
Ready for review and merge.

**Review by BigLep — APPROVED** [review]

**Pull request merged**